src/main/resources/site/static.ts

```diff
@@ -4,37 +4,30 @@ import type {
 } from '/index.d';
 
 
-import {toStr} from '@enonic/js-utils';
 // @ts-expect-error TS2307: Cannot find module '/lib/enonic/static' or its corresponding type declarations.
 import {buildGetter} from '/lib/enonic/static';
 import { GETTER_ROOT } from '/constants';
+import {DEBUG_MODE} from "/constants";
 
 
 const staticGetter = buildGetter({
-	etag: false, // default is true in production and false in development
+	etag: !DEBUG_MODE, // default is true in production and false in development
```

Comment:
Why?

Reply:
how can we ensure that people remember to set it to `true` in production?

Reply:
The comment is about what is default for the buildGetter function.

The immuteable getter should never have eTag. Always false.

Reply:
The eTag getter should always have etag: true